extra-dev/packages/coq-trakt/coq-trakt.dev/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "coq-trakt"
+maintainer: [ "Enzo Crance <enzo.crance@inria.fr>" "Louise Dubois de Prisque <louise.dubois-de-prisque@inria.fr>" "Chantal Keller <ckeller@lmf.cnrs.fr>" "Tomaz Mascarenhas <tomgm1502@gmail.com>" ]
+authors: [ "Enzo Crance" ]
+license: "LGPL-3.0-or-later"
+homepage: "https://github.com/ecranceMERCE/trakt"
+bug-reports: "https://github.com/ecranceMERCE/trakt/issues"
+dev-repo: "git+https://github.com/ecranceMERCE/trakt.git"
+doc: "https://ecrancemerce.github.io/trakt/"
+build: [ make "-j%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "coq-elpi" {>= "2.0.0"}
+  "coq" {>= "8.15~" }
+]
+tags: [ 
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "category:Miscellaneous/Coq Extensions"
+  "keyword:automation"
+  "keyword:elpi"
+  "date:2022-05-16"
```

Comment:
Having a release date in a package that follows a branch does not make much sense.